<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizerPNGReaderClass</class>
 <widget class="QWidget" name="VisualizerPNGReaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>310</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lidar Image Processing</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame1">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gbFrameControl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="title">
         <string>Image Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hsImageNb">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lImageName">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pbReadImage">
             <property name="text">
              <string>Read Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbLucky">
             <property name="text">
              <string>START</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbLoop">
             <property name="text">
              <string>LOOP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbLoopCount">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbEmpty">
             <property name="text">
              <string>Fill Buffer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pbReadContinuous">
             <property name="text">
              <string>Read Continuously</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbReadN">
             <property name="text">
              <string>Read N Images:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbReadN">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
